pages/reset/[token].jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { withRouter } from 'next/router'
+
+import styles from './styles.module.css'
+
+import { ChangePassword } from 'components/forms'
+
+const ResetStep2 = ({ router }) => (
+  <ChangePassword
```

Comment:
Do not you want to have a heading somewhere?

Reply:
What would you put there?

Reply:
Like _Create new password_ maybe

Reply:
Actually there is a headline inside that component. it's shared with settings section https://dashboard-git-reset-password.oacore.now.sh/reset/4545